styles/styles.css

```diff
@@ -738,6 +786,35 @@ main .section.light-blue {
     padding-right: 16px
   }
 
+/* START section - brand detail page */
+main .brand-detail-page img {
+  padding: 2.5rem 0;
+  width: 58.75rem;
+  max-width: 100%;
+  box-sizing: border-box;
+  display: block;
+  margin: 0 auto 2rem;
+}
+
+main .brand-detail-page .image-collage img {
+  width: unset;
+  max-width: unset;
+  padding: unset;
+  margin-bottom: 0.5rem;
+  zoom: 0.5;
+}
+
+main .brand-detail-page .image-collage.boxy-col-1::after {
+  content: unset;
+  flex: unset;
+}
+
+main .brand-detail-page p:has(img) {
```

Comment:
I think Firefox doesn't support `:has` out of the box
https://caniuse.com/?search=has

Reply:
Very good point 👍

Reply:
@dnbute thanks, this has been fixed